### PR TITLE
gixy: add page

### PR DESCRIPTION
**Pull request by Aracki**
I have added [gixy](https://github.com/yandex/gixy) - a tool for analyzing nginx configuration files.

### CODE REVIEW

File: pages/common/gixy.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# gixy
+
+> Analyze Nginx configuration.
+> More information: <https://github.com/yandex/gixy>.
+
+- Analyze nginx configuration (default path: /etc/nginx/nginx.conf):
```
Comment (pxgamer) [inline pages/common/gixy.md]: Should Nginx be stylised the same everywhere (i.e. Nginx, NGINX, nginx) for consistency?

File: pages/common/gixy.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`gixy`
+
+- Analyze nginx configuration but skip specific tests:
+
+`gixy --skips {{http_splitting}}`
+
+- Analyze nginx configuration with the specific level:
```
Comment (pxgamer) [inline pages/common/gixy.md]: Specific level of? :slightly_smiling_face:

File: pages/common/gixy.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`gixy {{-l|-ll|-lll}}`
+
+- Analyze nginx configuration on the specific path:
+
+`gixy {{config_path}}`
```
Comment (pxgamer) [inline pages/common/gixy.md]:
```suggestion
`gixy {{path/to/configuration_file}}`
```
Unless it's a config directory, in which case `configuration_directory` :slightly_smiling_face:

File: pages/common/gixy.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# gixy
+
+> Analyze nginx configuration.
```
Comment (sbrl) [inline pages/common/gixy.md]: Perhaps this will read better?
```suggestion
> Analyze nginx configuration files.
```

Comment (Aracki) [reply pages/common/gixy.md]: Singular is correct in this case.

Comment (sbrl) [reply pages/common/gixy.md]: That depends. Many configurations will have multiple nested configuration files in directories such as `sites-available` and `conf.d`. In addition, if we used the singular here we'd need to reword it to be `Analyze an nginx configuration file` to add an article, though this would imply that it can only check a single configuration file, @Aracki

Comment (Aracki) [reply pages/common/gixy.md]: Thanks, @sbrl for noticing this. I checked it, and it can check multiple files. So I modify the title and last example.
